go-controller/pkg/util/multi_network.go

```diff
@@ -279,7 +279,7 @@ func newLocalnetNetConfInfo(netconf *ovncnitypes.NetConf) (NetInfo, error) {
 
 	ni := &secondaryNetInfo{
 		netName:        netconf.Name,
-		topology:       types.Layer2Topology,
+		topology:       types.LocalnetTopology,
```

Comment:
nice catch, I guess we don't have tests yet for these parts which is why we didn't catch them yet? wondering if we have plans for some kind of UTs around these parts of code if not in this PR then at least maybe Miguel is writing them?

Reply:
Definitely should have UTs. I will discuss with Miguel to add this to the epic.